src/commands/createContainerApp/setQuickStartImage.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { IContainerAppContext } from "./IContainerAppContext";
+
+export function setQuickStartImage(context: IContainerAppContext): void {
+    context.image = 'mcr.microsoft.com/azuredocs/aci-helloworld';
```

Comment:
Minor nit, but the portal uses: 'mcr.microsoft.com/azuredocs/containerapps-helloworld:latest'

Wondering if we want to try and keep the quickstart image sources the same 🤔

Reply:
Yeah probably would be good to be consistent I'll change that!